src/components/layout/Header.vue

```diff
@@ -9,17 +9,17 @@
                     <router-link to="/" >
                         <img src="/theme/img/logo-w.png" class="header-logo"/>
                     </router-link>
-                
+
                     <div class="header-title">
                         Nem blockchain explorer
                     </div>
                     <div class="header-sub-title">
                         Search transactions, addresses, namespace & mosaic on the nem network.
                     </div>
-                    <SearchBox />
+                    <SearchBox class='search-box' />
                 </b-col>
                 <b-col md="3" class="header-left">
-                    <LanguageSelector class="d-none d-md-block"/>
+                    <LanguageSelector class="d-none d-md-block language-selector"/>
```

Comment:
seems like an unnecessary overhead to have "language-selector" wording doubled.

Reply:
language-selector is added as the className.
It use for **Cypress** to retrieve the the dom element.